scripts/scripts.js

```diff
@@ -38,6 +38,16 @@ const CONFIG = {
     sv: { ietf: 'sv', tk: 'fpk1pcd.css' },
   },
 };
+const prefersDarkScheme = window.matchMedia('(prefers-color-scheme: dark)');
+const userTheme = window.location.search.includes('theme=dark');
+
+function getTheme() {
```

Comment:
I'd argue this is more of a setter than a getter :)

Also, FYI, you could leverage ` document.body.classList.toggle('dark-theme', <bool>);` here to slightly reduce the code duplication.